djangocms_versioning_filer/test_utils/helpers.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+"""Module taken from filer repo tests.helpers
+As that can no longer be imported the path of least resistance was to copy it over
+"""
+
+from __future__ import absolute_import
+
+from filer.models.clipboardmodels import Clipboard, ClipboardItem
+from filer.models.foldermodels import Folder
+from filer.utils.compatibility import PILImage, PILImageDraw
+
+
+def create_superuser():
+    try:
+        from django.contrib.auth import get_user_model
+        User = get_user_model()
+    except ImportError:
+        from django.contrib.auth.models import User  # NOQA
+    superuser = User.objects.create_superuser('admin',
+                                              'admin@free.fr',
+                                              'secret')
+    return superuser
+
+
+def create_folder_structure(depth=2, sibling=2, parent=None):
+    """
+    This method creates a folder structure of the specified depth.
+
+    * depth: is an integer (default=2)
+    * sibling: is an integer (default=2)
+    * parent: is the folder instance of the parent.
+    """
+    if depth > 0 and sibling > 0:
+        depth_range = list(range(1, depth + 1))
```

Comment:
You should be able to avoid converting to list and reversing with a negative step value:

```
depth_range = range(depth, 0, -1)
```